extension/public/popup.html

```diff
@@ -0,0 +1,90 @@
+    <link rel="stylesheet" href="popup.css"/>
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+            <a>
+                <img src="./images/titled-logo-536x144.png"/>
+            </a>
+        </div>
+        <div class="version">
+            <span>Version 1.0.0</span>
+            <img src="./images/info-128.png" id="versionInfoBtn"/>
+        </div>
+    </header>
+    <div class="content">
+        <div class="top">
+            <div class="sites">
+                <h3>Enabled Sites</h3>
+                <div class="toggle-list" id="toggleList">
+                    <div class="toggle-item">
+                        <img src="./images/icon-reddit-128.png"/>
+                        <label class="switch">
+                            <input type="checkbox" id="redditSwitch">
+                            <span class="slider round"></span>
+                        </label>
+                    </div>
+                    <div class="toggle-item">
+                        <img src="./images/icon-imgur-128.png"/>
+                        <label class="switch">
+                            <input type="checkbox" id="imgurSwitch">
+                            <span class="slider round"></span>
+                        </label>
+                    </div>
+                    <div class="toggle-item">
+                        <img src="./images/icon-twitter-128.png"/>
+                        <label class="switch" >
+                            <input type="checkbox" id="twitterSwitch">
+                            <span class="slider round"></span>
+                        </label>
+                    </div>
+                    <button id="addOtherSites">Add other sites</button>
+                </div>
+                <div class="global">
+                    <img src="./images/icon-global-128.png"/>
+                    <label class="switch" >
+                        <input type="checkbox" id="globalSwitch">
+                        <span class="slider round"></span>
+                    </label>
+                </div>
+            </div>
+            <div class="keywords">
+                <h3>Keywords</h3>
+                <div class="btn-container">
+                    <button id="allowedBtn">Allowed</button>
+                    <button id="blockedBtn">Blocked</button>
+                </div>
+                <div class="tag-list" id="tagList">
+                </div>
+                <button class="manage-tag-list-btn" id="manageTagListBtn"">Manage</button>
+            </div>
+        </div>
+        <div class="bottom">
+            <div class="backend">
+                <span>Connect to our backend</span>
+                <label class="switch">
+                    <input type="checkbox" checked>
+                    <span class="slider round"></span>
+                </label>
+            </div>
```

Comment:
I think we should add a tooltip here to explain why they might want to connect to the backend vs not.
e.g.

**connecting:**
- shares your data
- reduces CPU load on your machine
- improves detection accuracy

Reply:
That would definitely be very useful and will improve UX.